uberwriter/markup_regex.py

```diff
@@ -21,9 +21,9 @@
 BLOCK_QUOTE = re.compile(
     r"^ {0,3}(?:> ?)+(?P<text>.+)", re.M)
 HEADER = re.compile(
-    r"^ {0,3}(?P<level>#{1,6}) (?P<text>[^\n]+)", re.M)
+    r"^ {0,3}(?P<level>#{1,6})(?P<text>[^\n]+)", re.M)
```

Comment:
I might be missing something, but this change breaks headers. At least Commonmark and Pandoc's Markdown (the 2 variants I usually test) require a space between the `#` and the header text. The tests are incorrect as well as they lack this space.